contrib/cirrus/test.sh

```diff
@@ -40,6 +40,21 @@ then
         unit)
             showrun make test-unit
             ;;
+        conformance)
+            case "$OS_RELEASE_ID" in
+            fedora)
+                warn "Installing moby-engine"
+                dnf install -y moby-engine
+                systemctl enable --now docker
+                ;;
+            ubuntu)
+                warn "Installing docker.io"
+                $LONG_APTGET install docker.io
+                systemctl enable --now docker
+                ;;
+            esac
```

Comment:
How about adding a `*)` case here with a failure message?

Reply:
Sure.

Reply:
setup.sh calls the `bad_os_id_ver` shell function for similar cases, so went with that.